hadoop-yarn-project/hadoop-yarn/hadoop-yarn-api/src/main/java/org/apache/hadoop/yarn/conf/YarnConfiguration.java

```diff
@@ -137,6 +137,16 @@ private static void addDeprecatedKeys() {
   // Resource types configs
   ////////////////////////////
 
+  public static final String NODE_STORE_ROOT_DIR_NUM_RETRIES =
+      YARN_PREFIX + "nodestore-rootdir.num-retries";
+
+  public static final int NODE_STORE_ROOT_DIR_NUM_DEFAULT_RETRIES = 3;
```

Comment:
Nit: To be inline with the other retries I think 500/1000 would be a more appropriate default.

Reply:
I will change it to 500